app/main/posts/views/share/post-export.directive.js

```diff
@@ -28,54 +29,85 @@ function PostExportController(
     ConfigEndpoint,
     Notify,
     $q,
-    PostFilters
+    PostFilters,
+    _
 ) {
     $scope.loading = false;
-    $scope.exportPosts = exportPosts;
-
-    function exportPosts() {
+    this.getQuery = getQuery;
+    this.prepareExport = prepareExport;
+    this.showCSVResults = showCSVResults;
+    this.requestExport = requestExport;
+    $scope.exportPostsConfirmation = function () {
```

Comment:
Based on the existing pattern this should become 

this.exportPostsConfirmation = exportPostsConfirmation;
function exportPostsConfirmation () {

though it might be handy to have a quick way to discern functions that are used by the template, function used only by the controller and functions used by both.

I'm kind of conflicted but for the sake of consistency changing it might be better. What do you think?

Reply:
I disagree that this would need to become this.exportPostsConfirmation , but agree that maybe we just want to be consistent here. And in that scenario, I would go with using $scope.exportPostsConfirmation instead of this.exportPostsConfirmation .